### PR TITLE
Use bindings v0.10.0

### PR DESCRIPTION
**Pull request by stefanobaghino-da**

### CODE REVIEW

File: daml.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-sdk-version: 0.13.55
+sdk-version: 1.0.0
 name: ex-tutorial-nodejs
 source: daml/PingPong.daml
 parties:
 - Alice
 - Bob
 exposed-modules:
```
Comment (cocreature) [inline daml.yaml]: I would recommend to remove the `exposed-modules` field. It already causes problems and warnings in some cases and not putting it there is a safe choice (it exposes everything).

**Review by cocreature — DISMISSED** [review]
Thanks!

File: daml.yaml (hunk 1 above)
Comment (stefanobaghino-da) [reply daml.yaml]: Thanks!

Comment (stefanobaghino-da) [reply daml.yaml]: Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/12/commits/524cd550328bfca79db7c7b6a6c6d0fb113ffdb9

**Comment by stefanobaghino-da** [comment]
@cocreature Sorry, my commit dismissed you review, I have to ask you to approve again.

**Review by cocreature — APPROVED** [review]
:facepalm:

**Pull request merged**